core/controllers/base_test.py

```diff
@@ -900,6 +900,19 @@ def test_alphabetic_i18n_keys(self):
                         self.log_line('Bad line in file: %s' % filename)
                 self.assertEqual(sorted(key_list), key_list)
 
+    # TODO(#14645): Remove this method when translation service is extended.
+    def test_hacky_tranlsation_keys_match_constants_en(self):
+        """Tests that the hacky translation keys present in constants file
+        are also present in en.json.
+        """
+        en_key_list = self._extract_keys_from_json_file('en.json')
+        hacky_translation_keys = constants.HACKY_TRANSLATION_KEYS
+        missing_hacky_translation_keys = list(
+            set(hacky_translation_keys) - set(en_key_list))
+        missing_hacky_translation_keys_length = len(
+            missing_hacky_translation_keys)
+        self.assertEqual(missing_hacky_translation_keys_length, 0)
```

Comment:
Maybe just use assertEqual instead (on the list)? Otherwise if this test fails you can't see at a glance what the issue is.

Reply:
Done!